### ISSUE TITLE
Token caching - different behaviour on iOS and Android

### ISSUE THREAD

**Issue by msardi23** [samcolby/react-native-ms-adal#8]
Firstly, thanks for providing this package. I have integrated this into an app with APIs protected by AzureAD and it works fine.

The issue I have observed in testing is with token caching. For example, on Android if I kill the app after authenticating and access it again the silent auth happens in the background as expected (i.e. I do not need to re-enter my credentials). However, on iOS, if I follow the same flow when I reopen the app I need to go through the process of entering my credentials again. Is this a known issue with iOS or is there something I have overlooked here in the setup steps needed on the iOS side?

**Comment by msardi23** [comment]
Something I overlooked...might be worth noting this in the iOS installation steps to aid people like me :)

> See the Microsoft Azure Active Directory Authentication Library (ADAL) for iOS and OSX for full instructions on how to configure the keychain etc in xcode.

**Comment by msardi23** [comment]
In relation to Android I observed some intresting behaviour in ensuring a user had truely logged out.

For example, I noticed that despite calling the logout function, when I went to login again my username was still displayed in the webview. In some cases, selecting the username resulted in successful authentication without me even needing to enter my password. 

To workaround this as part of my logout process I also clear cookies which seems to overcome the problem...more testing needed. I will close this issue.

` CookieManager.clearAll().then((res) => {...});`

This can be achieved using this package - [https://www.npmjs.com/package/react-native-cookies](url)
